src/docs.md

```diff
@@ -230,6 +230,18 @@ cadenzaClient.on('selectObjects:ok', (event) => {
 });
 ```
 
+### Change visibility of Layers in a Workbook Map
+
+<small>API: [CadenzaClient#setLayerVisibility](./classes/CadenzaClient.html#setLayerVisibility)</small>
+
+To change the visibility of a layer in an already shown map view, pass the layer path and the desired visibility.
+For layers in groups, the layer path contains the ancestors also. When making a layer visible, its ancestors will
+be made visible also. When hiding a layer, the ancestors are not affected. 
```

Comment:
Good hint :+1: Maybe rather add that to the API docs?

Reply:
ok